### DISCUSSION TITLE
Welcome to VisibleSim Forum!

### DISCUSSION THREAD

**Discussion by docben** [ProgrammableMatterProject/VisibleSim#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Psynge** [comment]
Hi, I am an Embedded Systems Engineer and a PhD student at the University of Massachusetts. I have been exploring various areas in Swarm Robotics and AI for a thesis topic. About a year ago a came up with what I thought was a novel platform, a quasi-spherical robot, made up of three orthogonal dodecagons with electromagnetic actuators on each face. I was a little disappointed to find the 3D Catoms already existed, which with slight differences, were exactly what I was proposing. Thankfully, there still is obviously much to be done in the field, and I hope to be able to contribute. I am planning to use VisualSim as a platform to develop algorithms useful for Swarm Robotic Self Assembly. 
Thank you for developing this simulation environment and making it available for everyone.
John O'Fallon

**Reply by julien-bourgeois** [reply]
Hi John,
Thanks for your comment!
Feel free to contact me on julien.bourgeois@univ-fcomte.fr if you want to discuss on the topic.
Julien
